src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -277,7 +282,9 @@ const BrokerFormContent: FC<Props> = ({
             {/* TODO: update `false` after cert support */}
             {getSchemaFromProtocol(formContent.protocol, false)}
```

Comment:
not necessarily part of the scope of this PR - but I saw this comment and remembered that its now applicable for us to address it. Feel free to move this to a new PR if you'd like, or include it here. 

```suggestion
            {getSchemaFromProtocol(formContent.protocol, isFlagEnabled('subscriptionsCertificateSupport') && formContent.authType === BrokerAuthTypes.Certificate)}
```